Page-10-Veg.html

```diff
@@ -0,0 +1,225 @@
+<!DOCTYPE html>
```

Comment:
Since page 10 has already been added,
You can fetch the latest changes from upstream , and change your page name to the latest page name (which has not been created yet)

Reply:
ok